### ISSUE TITLE
Can anybody say the reason for slack.rethinkdb shutdown ?

### ISSUE THREAD

**Issue by ajayashuba** [rethinkdb/rethinkdb#6626]
Was trying for long thought somebody would ask it

**Comment by nigel-dewar** [comment]
what is the link that is supposed to work?

**Comment by srh** [comment]
Slack.rethinkdb.com is supposed to work.

**Comment by ajayashuba** [comment]
http://slack.rethinkdb.com/     isn't  working for me though with error 'This site can’t be reached
'

**Comment by v3ss0n** [comment]
@srh it is not working for me too.

**Comment by sagivf** [comment]
This defiantly should be fixed, but perhaps we should consider moving to a different platform.
Reason is that the slack doesn't really have search, so any answers given there are basically lost which is pretty pretty pretty significant :)

**Comment by v3ss0n** [comment]
this is working : https://rethinkdb.slack.com

**Comment by sagivf** [comment]
@srh  So maybe just update the https://www.rethinkdb.com/community/ page

**Comment by nigel-dewar** [comment]
I know. SharePoint!!!! Hahahah. That was my joke for the year!

**Comment by nigel-dewar** [comment]
@v3ss0n @srh , yo guys can I please be added as a user for https://rethinkdb.slack.com

**Comment by v3ss0n** [comment]
you just register directly there. You got invite right?

**Comment by nigel-dewar** [comment]
Invite, that can only be given if some admin adds me to the slack group right? otherwise, Im not sure where or how you register???

![image](https://user-images.githubusercontent.com/9402456/36725244-692083c8-1c01-11e8-9ecb-1070698e5528.png)

**Comment by v3ss0n** [comment]
Eh so it seems the invite (bot?) is down.

**Comment by nigel-dewar** [comment]
Lol, no worries

**Comment by nigel-dewar** [comment]
In this case then, can any admin manually invite me so I can join?

**Comment by srh** [comment]
I think I'm going to start directing peolpe to this Discord channel now. https://discord.gg/PyQEsMD

**Comment by sagivf** [comment]
Discord seems nice!

**Comment by srh** [comment]
Still down.

**Comment by sourcepirate** [comment]
when i tried hitting rethinkdb.slack.com. I was able to join. Not sure whether it is offical or not.

**Comment by mxstbr** [comment]
:wave: We make a platform called [Spectrum.chat](https://spectrum.chat) which was born out of the need for a place for open source projects' communities to hang out! 

 **TL;DR: Spectrum is a mixture between Slack and a traditional forum, with all the benefits of real-time chat and of threaded conversation. It's born out of a need for a platform for communities like this one, and we'd love to have you!**

Contrary to Slack and Discord, all the conversations are public and threaded, so I can link you to [a previous conversation in e.g. the styled-components community](https://spectrum.chat/thread/956b1e5e-fe16-4991-a637-7a9457ac65df). That means as a maintainer, you don't end up answering the same question a million times over since people can find your answer on Google, or in the worst case you link back to it. It's real-time chat like you're used to from Slack though, so it still feels like there's "warm bodies" there if that makes sense.

Since it's threaded by default you wake up in the morning and don't have to try and decipher 500 new messages that were sent overnight, you look at the title of the 10 new threads that were posted and check out the one you're actually interested in. On top of that, it's a single login for all your communities—there's no need to create a new account for every. single. community. you. join, once you're on Spectrum you're on Spectrum, and can join all the communities on there.

We're also fully based on RethinkDB ourselves, using it for all our datastorage and real-time needs! You won't need to host your own invite bot that might or might not go down, it's all taken care of with 0 effort for the admins. We have a Slack invite tool, so moving over would just be a matter of an admin of the Slack team inviting all current members, and since we're big fans of open source and especially RethinkDB we'd obviously hook you up with a pro account.

Let me know what you think!

**Comment by mxstbr** [comment]
I went ahead and reserved https://spectrum.chat/rethinkdb, if y'all let me know who you want to own the community just tell me their username and I'll get them setup!

(also, feel free to join that and discuss all things RethinkDB anyway!)

**Comment by v3ss0n** [comment]
I was about to say this will split the community , but when i read made using rethinkdb , i am in.
We build chatrooms using rethinkdb + python(tornado) too.

**Comment by sagivf** [comment]
Seems Promising!! we do need proper search and this seems like an answer! Built with RethinkdDB is a huge plus.

**Comment by thelinuxlich** [comment]
and maybe a official forum using something like https://nodebb.org/

@srh Vue.js did this, and there is a bot that closes any issue not conforming to its standards, questions are directed to the forum

**Comment by mxstbr** [comment]
> Spectrum.chat seems a bit clunky in this stage.

We'll be open sourcing it soon once we find a bit of time to clean up the codebase, then you can help us make it less clunky! 😉

**Comment by sagivf** [comment]
I'm for supporting a fellow "thinker"! Getting some more innovative spirit into the mix will only help rethink evolve to.

**Comment by kasvith** [comment]
I also wanna join the official chat, i saw rethink yesterday and i felt love with it, sadly saw then it was shut down later....

**Comment by mxstbr** [comment]
Spectrum was just entirely open sourced a couple days ago! :tada: You can find the whole codebase here: https://github.com/withspectrum/spectrum

**Comment by thelinuxlich** [comment]
We should consider moving communication to spectrum then, update the website, we are losing momentum

**Comment by KrishnaPG** [comment]
@kasvith Everyone who comes across RethinkDB immediately appreciates the value it brings to the ecosystem. Being so unique in its offering, the biggest question is: why did it fail? What exactly went wrong?

I have seen criticism about other databases. But not this one. Studying and understanding the raise and fall of RethinkDB could be a great lesson for all of us open-source community members.

**Comment by krull** [comment]
#rethinkdb at freenode active?

**Comment by thelinuxlich** [comment]
Hey, listen!
In case anyone interested, there is a new community brewing around the fork of RethinkDB at https://spectrum.chat/rebirthdb

**Comment by nigel-dewar** [comment]
Thank you.

On 18 May 2018 4:35 AM, Alisson Cavalcante Agiani <notifications@github.com> wrote:

Hey, listen!
In case anyone interested, there is a new community brewing around the fork of RethinkDB at https://spectrum.chat/rebirthdb

**Comment by FilipeRamalho** [comment]
Could we get a Website update and a blog post on official communication ways ?

**Comment by thelinuxlich** [comment]
@FilipeRamalho we will have a new website up soon...and of course the repository will be available for everyone on the github org

**Comment by gabor-boros** [comment]
So, due to http://slack.rethinkdb.com is back, I'm closing this now.
